hack/builder/version.sh

```diff
@@ -1,3 +1,3 @@
-VERSION=30-7.0.0
+VERSION=30-8.0.2
```

Comment:
Quick question: I never found out what versioning schema we are using for this?

Reply:
There is none. I just bumped from 7 to 8. But there were various rebases, so I then changed the patch level to 1 and then 2 but I really just did something.